src/main/java/uk/gov/hmcts/cmc/claimstore/services/ResponseDeadlineCalculator.java

```diff
@@ -32,26 +29,14 @@ public ResponseDeadlineCalculator(
      * Calculates response deadline by date of issue.
      */
     public LocalDate calculateResponseDeadline(LocalDate issueDate) {
-        return calculateFirstWorkingDayAfterOffset(issueDate, serviceDays + timeForResponseInDays);
+        return issueDate.plusDays(timeForResponseInDays + serviceDays);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this addition operation to a "long". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS2184)